src/TableTransforms.jl

```diff
@@ -29,9 +29,10 @@ export
   colapply, colrevert,
 
   # built-in
+  Identity,
   Select,
   Reject,
-  Identity,
+  Rename3,
```

Comment:
This export is not correct. Also keep `Identity` in the line it was placed.

Reply:
> Also keep Identity in the line it was placed.
I was making it inline with the order of includes in `transforms.jl`